web-ui/src/pages/ReceivedRequestsPage.jsx

```diff
@@ -311,21 +294,15 @@ const ReceivedRequestsPage = () => {
         )}
         {!isLoading && (
           <div className="received-requests-container">
-            {receivedRequests.length === 0 && (
-              <div className="no-requests-message">
-                <Typography variant="body1">
-                  No received feedback requests
-                </Typography>
-              </div>
-            )}
-            {receivedRequests.length > 0 &&
-              filteredReceivedRequests.length === 0 && (
-                <div className="no-requests-message">
-                  <Typography variant="body1">
-                    No matching feedback requests
-                  </Typography>
-                </div>
-              )}
+            {receivedRequests.length === 0 ||
+              (receivedRequests.length > 0 &&
+                filteredReceivedRequests.length === 0 && (
```

Comment:
So close! you need to close the parens after the `filteredRecievedRequests.length === 0` check. Otherwise the "No matching feedback requests" will only show in the case that receivedRequests.length > 0, and you want it to show when it's === 0 as well.

Reply:
Lol I think I may have had that before but prettier auto removes it...

Reply:
Sorry for all the line changes, added a prettier fix for that for any future people working on this. The line you'll care about is 335